stactools_pipelines/pipelines/global_mangrove_watch/config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+  id: "global_mangrove_watch"
+  compute: "awslambda"
+  secret_arn: "arn:aws:secretsmanager:us-west-2:916098889494:secret:MAAP-STAC-auth-dev/MAAP-workflows-EsykqB" # cognito secret ARN
+  ingestor_url: "https://stac-ingestor.dit.maap-project.org"
+  inventory_location: "https://zenodo.org/records/6894273" # placeholder
```

Comment:
Still a placeholder?

Reply:
oh, nevermind - you're grabbing the urls from this webpage to build the inventory?